### PR TITLE
jupyter notebook demo for anomaly detector 

### PR DESCRIPTION
**Pull request by cabuliwallah**
#4029
add notebook demos for three anomaly detectors

### CODE REVIEW

**Comment by shane-huang** [comment]
1. Change the name of AIOps_anomaly_detect.ipynb in the same folder to `AIOps_anomaly_detect_unsupervised_forecast_based.ipynb. `
2. Change the name of your new notebook to `AIOps_anomaly_detect_unsupervised.ipynb`

**Comment by shane-huang** [comment]
Revise the README.md in the folder, add descriptions about two notebooks.

File: pyzoo/zoo/chronos/use-case/AIOps/README.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ We demonstrate how to use Chronos to detect anomaly data  based on historical ti
 In the reference use case, we use the publicly available cluster trace data cluster-trace-v2018 of Alibaba Open Cluster Trace Program. ([dataset link](http://clusterdata2018pubcn.oss-cn-beijing.aliyuncs.com/machine_usage.tar.gz)). 
  
 
-This use case example contains a notebook:
+This use case example contains two notebooks:
 
-- **AIOps_anomaly_detect.ipynb** demonstrates how to leverage Chronos's built-in models ie. MTNet, to do time series forecasting. 
+- **AIOps_anomaly_detect_unsupervised_forecast_based.ipynb** demonstrates how to leverage Chronos's built-in models ie. MTNet, to do time series forecasting. Then perform anomaly detection on predicted value with threshold detector.
```
Comment (shane-huang) [inline pyzoo/zoo/chronos/use-case/AIOps/README.md:11]: with threshold detector -> with ThresholdDetector(add links to api docs).

File: pyzoo/zoo/chronos/use-case/AIOps/README.md [hunk 2]
```diff
@@ -6,10 +6,11 @@ We demonstrate how to use Chronos to detect anomaly data  based on historical ti
...
 
+- **AIOps_anomaly_detect_unsupervised.ipynb** demonstrates how to perform anomaly detection based on Chronos's built-in dbscan detector, autoencoder detector and threshold detector.
```
Comment (shane-huang) [inline pyzoo/zoo/chronos/use-case/AIOps/README.md:13]: DBScanDetector(add links to api docs), AEDetector(add links to api docs) and ThresholdDetector(add links to api docs).

**Comment by shane-huang** [comment]
> 5. in notebook,
>    add "DBScanDetector assigns anomaly score 1 to anomaly samples, and 0 to normal samples" at the end of this paragraph.
> 
> > DBScanDetector uses DBSCAN clustering for anomaly detection. The DBSCAN algorithm tries to cluster the points and label the points that do not belong to any clusters as -1. It thus detects outliers detection in the input time series.
> 
> 1. some typos in below paragraph. Also make some changes as below:
> 
> > ThresholdDetector is a simple anomaly detector that detectes anomalies based on threshold. The target value for anomaly testing can be either 1) the sample value itself or 2) the difference between the forecasted value and the actual value. In this notebook we demostrate the first type. The thresold can be set by user or esitmated from the train data accoring to anomaly ratio and statistical distributions.

No. 5 and No. 6 in above is not addressed.

**Comment by cabuliwallah** [comment]
Jenkins passed:
http://10.239.176.111:18888/job/ZOO-PR-Validation/5773/

**Pull request merged**